lib/flagship.dart

```diff
@@ -64,8 +68,9 @@ class Flagship {
   ///
   /// visitorId : Id for the visitor
   /// context : Map that represent visitor's attribut  {"isVip":true}
-  static Visitor newVisitor(String visitorId, Map<String, Object> context) {
-    return Visitor(_configuration, visitorId, context);
+  static Visitor newVisitor(String visitorId, Map<String, Object> context,
+      {bool isConsent = true}) {
```

Comment:
isConsent is a bit weird, it would be like "estConsenti" is french. Maybe `hasConsented` instead ? as per the SDK specs file say

Reply:
initially i set **hasConsented**, and i change it recently .

Reply:
Fixed